examples/example-gateway/config/production.json

```diff
@@ -8,8 +8,10 @@
 
 	"metrics.tally.service": "example-gateway",
 	"metrics.m3.service": "example-gateway",
+
 	"tchannel.serviceName": "example-gateway",
 	"tchannel.processName": "example-gateway",
+	"tchannel.port": "7784",
```

Comment:
We dont have this port in production, we should allocate a tchannel port for edge-gateway. We can change this port number in the future.